addons/default/visiosoft/base-theme/resources/config/settings/sections.php

```diff
@@ -9,7 +9,7 @@
                 'fields' => [
                     'home_page_sub_categories_limit', 'navigation_title', 'navigation_action', 'date_fields',
                     'price_fields', 'breadcrumbs', 'ad_details', 'ad_details_tab', 'latest_and_view_all_btn',
-                    'register_page_instruction_logo', 'register_page_alert_link'
+                    'register_page_instruction_logo', 'register_page_alert_link', 'enabled_countries'
```

Comment:
we can say default country for register

Reply:
https://github.com/openclassify/openclassify/pull/715/commits/bd153541f906be1f2ca99f2bc06675828c464928